src/main/java/uk/gov/hmcts/ccd/domain/service/callbacks/CallbackService.java

```diff
@@ -91,8 +91,8 @@ public CallbackService(final SecurityUtils securityUtils,
             final HttpEntity requestEntity = new HttpEntity(callbackRequest, httpHeaders);
             return Optional.ofNullable(restTemplate.exchange(url, HttpMethod.POST, requestEntity, clazz));
         } catch (RestClientException e) {
-            LOG.info("Unable to connect to callback service {} because of {} {}", url, e.getClass().getSimpleName(), e.getMessage());
-            LOG.debug("", e);  // debug stack trace
+            LOG.warn("Unable to connect to callback service {} because of {} {}", url, e.getClass().getSimpleName(), e.getMessage());
+            LOG.warn("", e);  // debug stack trace
```

Comment:
probably debug for the stack trace? or you can mix with above log line itself.

Reply:
ok reverted to debug cheers